data/tlg0627/tlg001/tlg0627.tlg001.1st1K-grc1.xml

```diff
@@ -56,9 +55,9 @@
             <sourceDesc>
                 <biblStruct>
                     <monogr>
-                        <title>Oeuvres complètes d'Hippocrate</title>
+                        <title xml:lang="fre">Oeuvres complètes d'Hippocrate</title>
                         <author>Hippocrates</author>
-                        <editor>Émile Littré</editor>
+                        <editor xml:lang="fre">Émile Littré</editor>
```

Comment:
again, I wouldn't bother with the language for a modern name.

Reply:
Again, I was doing this because we have tagged modern German titles with the tag "deu". It was something I think that Matt and Annette started so I was trying to be consistent. :)

Reply:
Titles, yes, I get that. 
For names, there are only 60 or so of these — most are Latinized names.
There's one German dude and one Italian. 
I'm just saving some work here — as the language tag doesn't (necessarily) impart any crucial info in an editor name that I can tell. We aren't trying to parse the name or assign nationality. 
It's a good discussion to have and I'm happy to be persuaded it should be changed.